css/style.css

```diff
@@ -89,6 +87,48 @@ video {
   background-color: #003c8f;
 }
 
+/* Small hero */
+.small-hero__image--quote {
```

Comment:
@hanna446 I'm wondering if this could be named in a more descriptive way based on BEM. I believe the quote is actually the block, then the small hero would be the element and the background the modifier. (i.e. .quote__small-hero--background). This will impact the markup as well, of course. What do you think?